### PR TITLE
Updated homepage

### PR DESCRIPTION
**Pull request by jjessicas**
Added header image and header text to homepage

### CODE REVIEW

File: src/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -1,8 +1,66 @@
...
+
+const textStyle = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  fontFamily: "Work Sans, sans-serif",
+  color: "white",
+  fontSize: "65px", 
+  fontWeight: "600",
```
Comment (roshankern) [inline src/pages/HomePage/HomePage.js]:
```suggestion
  fontWeight: "400",
  textAlign: "center", 
```

Just making it a little less bold to match the figma design. Also, I think this text should be centered so it looks good when rescaled too.

**Review by roshankern — CHANGES_REQUESTED** [review]
Only one small suggestion.

Also, please fix the branch conflicts for this PR as Arohi demonstrated yesterday.

File: src/pages/HomePage/HomePage.js (hunk 1 above)
Comment (jjessicas) [reply src/pages/HomePage/HomePage.js]: Got it! I think i managed to fix the conflicts--I'll make a new PR for it.

**Comment by roshankern** [comment]
Looks good! I'll merge right now!

**Pull request merged**